cypress.config.js

```diff
@@ -6,6 +6,8 @@ module.exports = defineConfig({
         specPattern: 'cypress/tests/**/*.spec.{js,jsx,ts,tsx}',
         setupNodeEvents (on, config) {
             // implement node event listeners here
-        }
+        },
+        defaultCommandTimeout: 10000,
+        retries: { runMode: 3, openMode: 1 }
```

Comment:
is this required?

Reply:
It alleviates some of the flaky tests by retrying them. It's more of a band aid than a permanent fix but could provide some consistency when running e2e tests in CI

![image](https://github.com/user-attachments/assets/8087b5ab-e468-47df-8cb5-0fb9d7328214)

As can be seen in the logs, some tests that may have failed the first time passed the second or third. The downside is that it somewhat extends the duration of the test runner and accompanying costs